src/main/scala/org/allenai/deep_qa/experiments/datasets/Johannes.scala

```diff
@@ -0,0 +1,136 @@
+    ("sentence producer type" -> "background searcher") ~
+      ("searcher" -> corpus) ~
+      ("sentences" -> sentences) ~
+      ("sentence format" -> sentenceFormat)
+  }
+
+  val baseDir = "/efs/data/dlfa/turk_johannes_questions/"
+
+  // Train files
+  val trainFile = johannesFile(baseDir, "train")
+  val train = johannesDataset(baseDir, "train")
+  val mcTrainFile = makeMCFile(trainFile)
+  val mcTrainBuscBackgroundFile = makePassageBackgroundFile(
+    mcTrainFile,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+  val readingComprehensionTrainWithBuscBackgroundFile: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
+    ("sentences" -> mcTrainFile) ~
+    ("background" -> mcTrainBuscBackgroundFile)
+
+  val mcTrainLuceneBackgroundFile = makePassageBackgroundFile(
+    mcTrainFile,
+    "question and answer",
+    AristoDefaultCorpora.aristoDefaultElasticSearchIndex(3)
+  )
+  val readingComprehensionTrainWithLuceneBackgroundFile: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
+    ("sentences" -> mcTrainFile) ~
+    ("background" -> mcTrainLuceneBackgroundFile)
+
+  // Dev Files
+  val devFile = johannesFile(baseDir, "dev")
+  val dev = johannesDataset(baseDir, "dev")
+  val mcDevFile = makeMCFile(devFile)
+  val mcDevBuscBackgroundFile = makePassageBackgroundFile(
+    mcDevFile,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+  val readingComprehensionDevWithBuscBackgroundFile: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
+    ("sentences" -> mcDevFile) ~
+    ("background" -> mcDevBuscBackgroundFile)
+
+  val mcDevLuceneBackgroundFile = makePassageBackgroundFile(
+    mcDevFile,
+    "question and answer",
+    AristoDefaultCorpora.aristoDefaultElasticSearchIndex(3)
+  )
+  val readingComprehensionDevWithLuceneBackgroundFile: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
+    ("sentences" -> mcDevFile) ~
+    ("background" -> mcDevLuceneBackgroundFile)
+
+  // Test Files
+  val testFile = johannesFile(baseDir, "test")
+  val test = johannesDataset(baseDir, "test")
+  val mcTestFile = makeMCFile(testFile)
+  val mcTestBuscBackgroundFile = makePassageBackgroundFile(
+    mcTestFile,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+  val readingComprehensionTestWithBuscBackgroundFile: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
+    ("sentences" -> mcTestFile) ~
+    ("background" -> mcTestBuscBackgroundFile)
+
+  val mcTestLuceneBackgroundFile = makePassageBackgroundFile(
+    mcTestFile,
+    "question and answer",
+    AristoDefaultCorpora.aristoDefaultElasticSearchIndex(3)
+  )
+  val readingComprehensionTestWithLuceneBackgroundFile: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
+    ("sentences" -> mcTestFile) ~
+    ("background" -> mcTestLuceneBackgroundFile)
+
+
+  // Datasets
```

Comment:
If you're grouping it like this, you should probably move the `train`, `dev`, and `test` lines down here.